app/javascript/components/ReservationForm.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import DatePicker from 'react-datepicker';
+import "react-datepicker/dist/react-datepicker.css";
+
+const ReservationForm = () => {
+ const [date, setDate] = useState(new Date());
+ const [city, setCity] = useState('');
+
+ const handleSubmit = (event) => {
+  event.preventDefault();
+  // Make a POST request to the Rails API to create the reservation
+ };
+
+ return (
+  <form onSubmit={handleSubmit}>
+    <label>
+      Date:
+      <DatePicker selected={date} onChange={(date) => setDate(date)} />
+    </label>
+    <label>
+      City:
+      <input type="text" value={city} onChange={(event) => setCity(event.target.value)} />
+    </label>
+    <input type="submit" value="Reserve" />
+  </form>
```

Comment:
<img width="874" alt="Capture d’écran 2023-12-12 à 08 59 47" src="https://github.com/MenendezON/book-an-appointment/assets/5085442/77546c41-ba42-4939-b45d-9d323914be30">
Wonderful, I appreciate your concern about this project. According to the requirements, there are two ways to make a reservation, so this form doesn't take into account the 2nd way. Please kindly add a new field for the item(motorbike)